app/mailers/order_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class OrderMailer < ApplicationMailer
+  default from: "receipt@bookstore.com"
```

Comment:
i _think_ people recommend using either an SB email or example.com address in case emails are accidentally sent out